src/plugins/python/pythonplugin.cpp

```diff
@@ -192,7 +227,7 @@ void PythonPlugin::reloadModules()
 /**
  * Finds the first Python class that extends tiled.Plugin
```

Comment:
Comment needs adjusting.
```suggestion
 * Finds the first Python class that extends the given \a pluginClass.
```